### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by jglassenberg**
Adding the Agenda Maker script sample

### CODE REVIEW

File: Agenda Maker/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+First, create the script:
+
+1.  [Click here][code] to open the code in the Apps Script code editor.
+2.  Click the menu item **File > Make a copy** to get your own copy. You can
+    later access this copy in the [G Suite Developer Hub][hub].
+
+[code]: https://github.com/gsuitedevs/solutions/tree/master/Agenda%20Maker/src/code.js
+[hub]: https://script.google.com
+
```
Comment (asrivas) [inline Agenda Maker/README.md:46]: nit: extra new line

File: Agenda Maker/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## Try it
+
+First, create the script:
+
+1.  [Click here][code] to open the code in the Apps Script code editor.
+2.  Click the menu item **File > Make a copy** to get your own copy. You can
+    later access this copy in the [G Suite Developer Hub][hub].
+
+[code]: https://github.com/gsuitedevs/solutions/tree/master/Agenda%20Maker/src/code.js
```
Comment (asrivas) [inline Agenda Maker/README.md:44]: apologies in advance for being a sticker but can you capitalize Code.js like the in the template? I can't remember if our automation expects but better to be safe than sorry!

File: Agenda Maker/src/code.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+  return folder.getId();
+}
+
+
+/*
+Find the template agenda doc, or create a default if it doesn't exist
+*/
+function getTemplateId(folderId){
+
```
Comment (asrivas) [inline Agenda Maker/src/code.js:28]: nit: extra new line

File: Agenda Maker/src/code.js [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    var file = files.next();
+    return file.getId();
+  }
+  
+  
+  //Otherwise, create the agenda template.  
+  // Feel free to adjust the default template here
+  var doc = DocumentApp.create('Agenda TEMPLATE##');
+  var body = doc.getBody();
+  //var rowsData = [['Plants', 'Animals'], ['Ficus', 'Goat'], ['Basil', 'Cat'], ['Moss', 'Frog']];
```
Comment (asrivas) [inline Agenda Maker/src/code.js:43]: Remove

File: Agenda Maker/src/code.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+      
+      //Replace the tag with a link to the agenda document
+      var agendaUrl = "https://docs.google.com/document/d/" + newDoc.getId();
+      description = description.replace("##agenda", "<a href=" + agendaUrl + ">Agenda Doc</a>");
+      event.setDescription(description);
+      
+      //Invite attendees to the Google doc so they automatically receive access to the agenda
+      newDoc.addEditor(newDoc.getOwner());
+
+      //Logger.log("event added: "+event.getTitle()+description);
```
Comment (asrivas) [inline Agenda Maker/src/code.js:141]: Remove

File: Agenda Maker/src/code.js [hunk 6]
```diff
@@ -0,0 +1,146 @@
+
+/*
+See if the folder of Agenda docs exists, or otherwise create it.
+*/
+function checkFolder (){
+  var folders = DriveApp.getFoldersByName("Agenda Maker - App");
+  //Find the folder if it exists
+  while(folders.hasNext()){
+    var folder = folders.next();
+    if ((folder.getDescription() == "App Script App - Do not change this description") && (folder.getOwner().getEmail()==Session.getActiveUser().getEmail()))
```
Comment (asrivas) [inline Agenda Maker/src/code.js:10]: *Apps

File: Agenda Maker/src/code.js [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+        body.replaceText("##Attendees##", "Attendees");
+      }
+      
+      //Replace the tag with a link to the agenda document
+      var agendaUrl = "https://docs.google.com/document/d/" + newDoc.getId();
+      description = description.replace("##agenda", "<a href=" + agendaUrl + ">Agenda Doc</a>");
+      event.setDescription(description);
+      
+      //Invite attendees to the Google doc so they automatically receive access to the agenda
+      newDoc.addEditor(newDoc.getOwner());
```
Comment (asrivas) [inline Agenda Maker/src/code.js:139]: does only add the owner? Could you save the attendees in the for loop above and add them as viewers or editors?

File: Agenda Maker/src/code.js [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+      //Logger.log("event added: "+event.getTitle()+description);
+    }
+  }
+  //}
```
Comment (asrivas) [inline Agenda Maker/src/code.js:144]: Remove

File: Agenda Maker/src/code.js (hunk 6 above)
Comment (jglassenberg) [reply Agenda Maker/src/code.js]: Can you clarify the requested change for this line of code?

**Pull request merged**